neurom/fst/_mm.py

```diff
@@ -101,11 +101,6 @@ def map_sections(fun, nrn, neurite_type=NeuriteType.all):
                 for s in i_chain2(nrn.neurites, tree_filter=is_type(neurite_type)))
 
 
-def section_lengths(nrn, neurite_type=NeuriteType.all):
-    '''section lengths'''
-    return map_sections(mm.path_distance, nrn, neurite_type)
```

Comment:
This is a bit confusing, or maybe I don't understand it correctly. The path_distance is not suppose to return the same as the path_distance. The first is the sum(length of all segments in a section), the second is sum(length of all segments from current section to the root of the tree).